src/core/components/courses/ProfileOrganization.vue

```diff
@@ -161,10 +168,15 @@ export default {
         this.courseLoading = false;
       }
     },
-    async refreshTrainers () {
+    async refreshTrainersAndSalesRepresentatives () {
       try {
         const trainers = await Users.list({ role: [TRAINER, TRAINING_ORGANISATION_MANAGER, VENDOR_ADMIN] });
-        this.trainerOptions = formatAndSortIdentityOptions(trainers);
+
+        const [trainerRole] = await Roles.list({ name: [TRAINER] });
+        const salesRepresentatives = trainers.filter(t => t.role.vendor !== trainerRole._id);
+
+        this.trainerOptions = Object.freeze(formatAndSortIdentityOptions(trainers));
```

Comment:
pourquoi ce rajout du Object freeze ?

Reply:
Il me semble que c'est une bonne pratique (on en avait discuté avec Sophie). Je ne suis pas sûr que ça soit très utile ici sachant que ce n'est pas une longue liste mais j'ai voulu essayer de le mettre pour tester :
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze
https://vuedose.tips/improve-performance-on-large-lists-in-vue-js/

Reply:
je pense que c'est une bonne habitude a prendre :)